test/integration/actions.test.js

```diff
@@ -112,14 +116,14 @@ test('create, get and delete with parameters an action', t => {
   }).catch(errors)
 })
 
-test('invoke action with fully-qualified name', t => {
+tempTest('invoke action with fully-qualified name', t => {
```

Comment:
Has this been skipped because that action is not installed on the integration version? I'd prefer to work out a way to make this test work rather than skipping. We could install the `whisk.system/utils` package during setup on the integration host or re-write the test to invoke another available action using the fully qualified name. The former seems preferable?